GovUkDesignSystem/HtmlGenerators/DateInputHtmlGenerator.cs

```diff
@@ -67,6 +68,8 @@ internal static class DateInputHtmlGenerator
                 labelOptions.For = propertyId;
             }
 
+            var disabledAttributes = disabled ? new Dictionary<string, string> {{"disabled", "true"}} : null;
```

Comment:
As we just need "disabled" in the tag rather than "disabled="true"" this can be:
`new Dictionary<string, string> {{"disabled", null}}`